### PR TITLE
Integrate font & display plugins

### LINKED ISSUE #112: Add `postcss-normalize-display-values`
https://github.com/cssnano/cssnano/tree/master/packages/postcss-normalize-display-values

> Normalize display property values with PostCSS.
> ### Input
> 
> ```css
> div {
>     display: inline flow-root
> }
> ```
> 
> ### Output
> 
> ```css
> div {
>     display: inline-block
> }
> ``` 

Spec: https://developer.mozilla.org/en-US/docs/Web/CSS/display

### LINKED ISSUE #116: Add postcss-font-format-keywords
Seems like [postcss-font-format-keywords](https://github.com/valtlai/postcss-font-format-keywords) provides a syntax sugar from the CSSWG spec. Should we add it to polyfills pack?

cc @valtlai

### PR DESCRIPTION
**Pull request by Antonio-Laguna**
Fixes #116 
Fixes #112

### CODE REVIEW

File: cli/csstools-cli/CHANGELOG.md [hunk 1]
```diff
@@ -3,12 +3,16 @@
 ### Unreleased (minor)
 
 - Emit warnings from plugins
-- Added `@csstools/postcss-is-pseudo-class`
+- Added `@csstools/postcss-font-format-keywords`
```
Comment (romainmenke) [inline cli/csstools-cli/CHANGELOG.md:6]: This also needs a change in `cli/csstools-cli/src/cli.ts`

1. add the plugin in the `switch` statement
2. add the key from the `switch` statement in the general help output

The general help also exists in the README, but this doesn't always have to be up to date as it's an example

**Review by romainmenke — COMMENTED** [review]
A few small things for the CLI

File: cli/csstools-cli/CHANGELOG.md (hunk 1 above)
Comment (Antonio-Laguna) [reply cli/csstools-cli/CHANGELOG.md:6]: Thanks for reviewing that! I've added those!

**Review by romainmenke — APPROVED** [review]
LGTM!

**Pull request merged**